packages/tools/addon/styles/cs-field-overlays.css

```diff
@@ -37,3 +42,15 @@
 .cs-field-overlay.passthrough > * {
   pointer-events: auto;
 }
+
+/* Overriding style attribute that sets `visibility` to `hidden`.
+   This is so that when user can exit focus on field by
+   clicking elsewhere on the page */
+.cs-field-overlay.focused > .liquid-child {
+  visibility: visible !important;
```

Comment:
This is likely to cause animations to flicker. The liquid-child is only supposed to be hidden temporarily, during the start of an animation. If it's staying hidden, that's a bug, and we should fix that instead.

Reply:
Yes, something is off with the scrim behavior, I'll investigate further.